syntax = "proto3";
package jio.brain.proto.knowledge.healthcare;

import "jio/brain/proto/knowledge/healthcare/base/search_response.proto";
import "jio/brain/proto/knowledge/healthcare/base/status.proto";

message GetDiseasePatientStateRequest {
    uint32 disease_id = 1;
}

message GetDiseasePatientStateResponse { 
    repeated uint32 gender = 1;
    uint32 minimum_age = 2;
    uint32 maximum_age = 3;
    jio.brain.proto.knowledge.healthcare.base.Status status = 4;
}